ratis-server/src/main/java/org/apache/ratis/server/storage/RaftStorageDirectoryImpl.java

```diff
@@ -150,6 +152,13 @@ StorageState analyzeStorage(boolean toLock) throws IOException {
       this.lock(); // lock storage if it exists
     }
 
+    // check enough space
+    if (!hasEnoughSpace()) {
+      LOG.warn("There are not enough space left for directory " + rootPath
+          + "required: " + spaceReserved + " left: " + root.getFreeSpace());
```

Comment:
Let's use "free space" and add some comma as below
`", required: " + spaceReserved + ", free space: " + root.getFreeSpace());`

Reply:
OK.